max_particles: 1000
min_particles: 1000
# 运动超过该范围则更新粒子
update_dist: 0.1
update_angle: 0.1
# scan 的最大扫描距离
max_range: 10.0
publish_odom: true
max_viz_particles: 300

range_method: "cddt"
theta_discretization: 360
rangelib_variant: 2
downsample_size: 23
motion_dev_x: 0.3
motion_dev_y: 0.2
motion_dev_theta: 0.5
squash_fac: 4

laser_frame_id: "/laser"
odom_frame_id: "/odom"
base_link_frame_id: "/base_link"
map_frame_id: "/map"

# 雷达自身的参数，误差
z_short: 0.01
z_max: 0.07
z_rand: 0.12
sigma_hit: 8.0
z_hit: 0.5
c_r: 0.01
